db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
-Dir[Rails.root.join("db/seeds/dummy_structures.rb")].each { |seed| load seed }
+Dir[Rails.root.join("db/seeds/initial_seed.rb")].each { |seed| load seed }
```

Comment:
Way easier to just require / require_relative the files. It's not as if there's loads of them

Reply:
I blame @kicferk1 😛

Reply:
Would you like this changed, or are you happy to leave this?

Reply:
Change it please.